src/main/java/org/fundacion/jala/converter/models/AuthenticationResponse.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ *
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ */
+package org.fundacion.jala.converter.models;
+
+public class AuthenticationResponse {
+    private final String jwt;
+
+    public AuthenticationResponse(String jwt) {
```

Comment:
Your parameters in some methods should be final and shoul not have the same name as the variable to which it is being assigned

Reply:
I do not agree on the variable's names, but I changed them.